wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/document/DocumentInfoScreen.kt

```diff
@@ -246,24 +255,29 @@ fun DocumentInfoScreen(
                     text = { Text(text = stringResource(R.string.document_info_screen_menu_item_delete)) },
                     leadingIcon = { Icon(Icons.Outlined.Delete, contentDescription = null) },
                     onClick = {
-                        showMenu = false
+                        showMenu = false;
                         showDeleteConfirmationDialog = true
```

Comment:
Looks like work-in-progress edits you forgot to remove?

Reply:
Nah, I thought it was needed, but looks like it is not. Removed.